src/clj/game/cards-assets.clj

```diff
@@ -428,6 +428,12 @@
                                          (shuffle! state side :deck))}
                            card nil))}]}
 
+   "Jeeves Model Bioroids"
+   {:abilities [{:label "Gain [Click]"
+                 :req (req (< 2 (- (:click-per-turn corp) (:click corp))))
```

Comment:
Also the common combo with Jeeves is Sublimal Messages with Shipment from SanSan, which would also not work with this. Maybe we should just remove the req for now and leave it to the player to decide when to use Jeeves to get the click? I don't know how much work it would be to fully automate Jeeves